src/components/spinner/styled.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import { Colour, theme } from '@fellesdatakatalog/theme';
+import spinnerImage from '../../images/spinner-digdir-fdk.svg';
+
+const Spinner = styled.div`
+  border: 16px solid ${theme.colour(Colour.NEUTRAL, 'N50')};
+  border-top: 16px solid ${theme.colour(Colour.CYAN, 'C50')};
+  border-radius: 50%;
+  width: 120px;
+  height: 120px;
+`;
+
+const SpinnerImage = styled.div`
```

Comment:
Om ikke du trenger å gjøre noen endringer på hvordan bildet vises trenger du ikke å wrappe dette i et element. Du kan bare gjøre `import SpinnerImage from '[...].svg'` og `<SpinnerImage/>` i JSX. Søk på ".svg" for inspirasjon

Reply:
Hmmm.. det var @oyvindis som sa at jeg skulle bruke styled components. Men hva tenker dere, skal jeg ta det bort?

Reply:
Det er litt avhengig av hvordan det skal se ut. Trenger du ikke å endre på svg så kan du legge direkte inn. Ser du har bredde og høyde på elementet over, det ville jeg kanskje hatt direkte på bildet.

Reply:
Ok dere, da har jeg lagt bredde og høyde på bildet i stedet for på div'en. Er dette ok? Er det mer som trengs å gjøres nå? :innocent:

Reply:
Lagt det i selve `.svg` fila @frederik-ronnevig ? Hvis du vil treffe et underelement i CSS kan du skrive for eksempel:

```suggestion
const SpinnerImage = styled.div`
    & > svg {
        width: 120px;
        height: 120px
        }
    `
```

Reply:
Dette cheat sheetet har veldig god oversikt over CSS selectors, bruker det hele tiden selv :) 
http://tutorials.jenkov.com/css/selectors.html

Reply:
Å ja, nå skjønner jeg hva du mente med den første kommentaren. Jeg prøver det.

Reply:
Sånn, nå skal dette være i orden! :-) Håper det er tilfredsstillende. :-)